src/app/collection-detail-etb/collection-detail-etb.page.ts

```diff
@@ -1354,54 +1352,24 @@ export class CollectionDetailEtbPage implements OnInit {
   }
 
   playContent(event) {
-    this.headerService.hideHeader();
+    const telemetryDetails = {
```

Comment:
Why page Id is COURSE_DETAILS in Collection page ?

Reply:
Was a mistake, will change to "COLLECTION_DETAIL"